### PR TITLE
Feature/14 filter by users

### PR DESCRIPTION
**Pull request by al-ros**
image

Working user filter feature

Changes: 

App.js:
1. add filterValue state
2. add handleFilterChange

pass new props to Header and Calendar

### CODE REVIEW

File: src/components/ModalAuth/ModalAuth.jsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ const ModalAuth = ({ open, onSubmit }) => {
       <Dropdown options={ USERS } 
         value={ user }
         onChange={ ({target: { value}}) => setUser(value) } />
+        {/* {console.log(user)} */}
```
Comment (roman-vabishchevych) [inline src/components/ModalAuth/ModalAuth.jsx]: please remove

File: src/components/Calendar/Calendar.jsx [hunk 2]
```diff
@@ -25,6 +26,7 @@ const Calendar = ({onClickEditEvent}) => {
                     </div>)}
                 </div>
             ))}
+            {/* {console.log(user)} */}
```
Comment (roman-vabishchevych) [inline src/components/Calendar/Calendar.jsx]: remove plz

File: src/App.js [hunk 3]
```diff
@@ -10,11 +10,13 @@ import Calendar from './components/Calendar';
 const USERS = [
   { label: 'Admin', value: 'admin', role: 'admin' },
   { label: 'Sasha', value: 'sasha', role: 'user' },
-  { label: 'Roma', value: 'roma', role: 'user' }
+  { label: 'Roma', value: 'roma', role: 'user' },
+  { label: 'All users', value: 'allUsers', role: 'allUsers'}
```
Comment (roman-vabishchevych) [inline src/App.js]: all users – is out of the scope, so please remove it

**Review by roman-vabishchevych — CHANGES_REQUESTED** [review]

File: src/components/ModalAuth/ModalAuth.jsx (hunk 1 above)
Comment (al-ros) [reply src/components/ModalAuth/ModalAuth.jsx]: done

File: src/components/Calendar/Calendar.jsx (hunk 2 above)
Comment (al-ros) [reply src/components/Calendar/Calendar.jsx]: done

File: src/App.js (hunk 3 above)
Comment (al-ros) [reply src/App.js]: done

**Review by roman-vabishchevych — APPROVED** [review]

**Pull request merged**